packages/react/src/components/login/sessionPoller/hooks.test.tsx

```diff
@@ -0,0 +1,211 @@
+import HttpStatusCode from 'http-status-typed';
+import { disableFetchMocks, enableFetchMocks } from 'jest-fetch-mock';
```

Comment:
This is weird. The package is listed in package.json and this error does not show in my local env.  And the project is built here just fine. Have you ran yarn install when switching branches? Your IDE seems to warn about all new packages, so that might be it.

Reply:
See [discussin](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1042#discussion_r1363685512)